test/components/ChartForm_test.js

```diff
@@ -49,25 +42,59 @@ describe('ChartForm.vue', () => {
     it('creates the expected date field options', () => {
       const dateFieldOptions = wrapper.findAll(`${selector.dateFieldSelect} > option`);
       expect(dateFieldOptions.length).toEqual(5); // 4 options plus prompt
-      expect(dateFieldOptions.at(1).text().trim()).toEqual('enroll_date (Enrollment date)'); // sorted by label
+      expect(
+        dateFieldOptions
+          .at(1)
+          .text()
+          .trim()
+      ).toEqual('enroll_date (Enrollment date)'); // sorted by label
     });
 
     it('creates the expected date interval options', () => {
-      const dateIntervalOptions = wrapper.findAll(`${selector.dateIntervalSelect} > option`);
+      const dateIntervalOptions = wrapper.findAll(
+        `${selector.dateIntervalSelect} > option`
+      );
       expect(dateIntervalOptions.length).toEqual(5); // 4 options plus prompt
-      expect(dateIntervalOptions.at(1).text().trim()).toEqual('Days');
-      expect(dateIntervalOptions.at(2).text().trim()).toEqual('Weeks');
-      expect(dateIntervalOptions.at(3).text().trim()).toEqual('Months');
-      expect(dateIntervalOptions.at(4).text().trim()).toEqual('Years');
+      expect(
+        dateIntervalOptions
+          .at(1)
+          .text()
+          .trim()
+      ).toEqual('Days');
```

Comment:
This code certainly looks more verbose now. Seems like if the whole call chain can fit within the 90 columns then it shouldn't be broken up.

Reply:
This definitely doesn't look as good. On the other hand, this pattern is kind of a smell; it would be better handled by writing a utility that wraps the `at().text().trim()` chain.

Reply:
Yeah, I think seeing this would prompt me to refactor, which is probably a good thing.